modules/odc-grouping-multiple-components.adoc

```diff
@@ -11,20 +11,28 @@ The following procedure adds a MariaDB database service to an existing applicati
 
 .Prerequisites
 
-* Ensure that you have created and deployed a Node.js application on {product-title} using the *Developer* perspective.
+* You have created and deployed minimum two or more components on {product-title} using the *Developer* perspective.
 
 .Procedure
 
-. Create and deploy the MariaDB service to your project:
-.. In the *Developer* perspective, navigate to the *+Add* view and select the *From Catalog* option to view all available services in the *Developer Catalog*.
-.. Select *Databases* to list all the database services and then click *MariaDB* option to see the details for the service.
-.. Click *Instantiate Template* to see an automatically populated template with details for the MariaDB service, and then click *Create* to create and view the MariaDB service in the *Topology* view.
+* To add a service to the existing application group, press kbd:[Shift]+ drag it to the existing application group. Also this will add the required labels to the component.
 
-. To add the MariaDB service to the existing application group, press kbd:[Shift]+ drag the *mariadb* pod to the application group that contains the Node.Js component. The MariaDB service is added to the existing application group.
+[NOTE]
+====
+Dragging a component and adding it to an application group adds the required labels to the component.
```

Comment:
You mention: Also this will add the required labels to the component.
Why do we need a note in addition? Also an image in a note is awkward.

Reply:
Suggest adding "Dragging a component and adding it to an application group adds the required labels to the component." to the step instead of "Also this will add the required labels to the component."
And add the image after the step.